index.html

```diff
@@ -235,11 +235,11 @@ <h6 class="text-uppercase fw-bold mb-4">Contact</h6>
             <p><i class="fas fa-home me-3 text-secondary"></i> Kandy, Sri Lanka</p>
             <p>
               <i class="fas fa-envelope me-3 text-secondary"></i>
-              <a href="mailto:carexnextgen@gmail.com" class="text-info">carexnextgen@gmail.com</a>
+              <a href="mailto:carexnextgen@gmail.com" class="e-mail">carexnextgen@gmail.com</a>
```

Comment:
Use bootstrap for this @SilunaP brother.

- Remove the text-info and put text-light

Reply:
ok

Reply:
👍